lua/weapons/arccw_base/cl_hud.lua

```diff
@@ -276,6 +278,20 @@ function SWEP:DrawHUD()
                 apan_bg.x = toscreen.x - apan_bg.w - ScreenScaleMulti(8)
                 apan_bg.y = toscreen.y - apan_bg.h * 0.5
 
+				if GetConVar("arccw_hud_3dfun_ammotype"):GetBool() then
+					local wammotype = {
+						x = apan_bg.x + apan_bg.w - airgap,
+						y = apan_bg.y - ScreenScaleMulti(8),
+						text = tostring(data.ammotype),
```

Comment:
Consider using `language.GetPhrase()` to print the nice name instead of the raw name?